.gitlab/package_build/deb.yml

```diff
@@ -19,6 +19,10 @@
     - chmod 0744 /tmp/system-probe/clang-bpf /tmp/system-probe/llc-bpf
     - inv -e omnibus.build --release-version "$RELEASE_VERSION" --major-version "$AGENT_MAJOR_VERSION" --python-runtimes "$PYTHON_RUNTIMES" --base-dir $OMNIBUS_BASE_DIR ${USE_S3_CACHING} --skip-deps --go-mod-cache="$GOPATH/pkg/mod" --system-probe-bin=/tmp/system-probe ${FLAVOR:+--flavor $FLAVOR}
     - ls -la $OMNIBUS_PACKAGE_DIR
+    - curl -sSL "https://dd-package-tools.s3.amazonaws.com/dd-pkg/${DD_PKG_VERSION}/dd-pkg_Linux_${DD_PKG_ARCH}.tar.gz" | tar -xz -C /usr/local/bin dd-pkg
```

Comment:
should we install this directly in the docker buildimages?
And even if it's internal, should we do a checksum verification before extraction?

Reply:
Spencer can confirm, but I think for now we want to install it here to reduce the friction for updating to a new version of the tool.

Reply:
Correct, we want to default to using the `latest` release - and updating the buildimages every time we release seemed less ideal. If we end up stabilizing or getting feature complete with the tooling moving it to the image itself seems reasonable.

Regarding checksums, we don't do so today but could add it in the future. I'd like to get this initial step in place so we can show progress on the SRR/OKRs and continue to add polish as we iterate on the tooling.